x/evm/types/chain_config.go

```diff
@@ -141,6 +145,12 @@ func (cc ChainConfig) Validate() error {
 	if err := validateBlock(cc.MergeNetsplitBlock); err != nil {
 		return errorsmod.Wrap(err, "MergeNetsplitBlock")
 	}
+	if err := validateBlock(cc.ShanghaiBlock); err != nil {
```

Comment:
can you add tests for these too?

Reply:
I will open a new PR for the 2 new blocks and the tests.